ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -530,7 +530,7 @@ impl AotIRDisplay for Instruction {
                 binop.to_string(m),
                 rhs.to_string(m)
             )),
-            Self::Br => ret.push_str("br"),
+            Self::Br { .. } => ret.push_str("br"),
```

Comment:
Can you include the new operand in the stringified instruction?

Reply:
[11ce067](https://github.com/ykjit/yk/pull/1115/commits/11ce067b6c840c08332b4c99b63137277d90eeba)